### PR TITLE
Confirmation Message on Form Submission

### PR DESCRIPTION
**Pull request by talyh**
Form submission shows confirmation message

**Coming up next**
Redux
Session persistency (stored token validation)

### CODE REVIEW

File: .firebase/hosting.YnVpbGQ.cache [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+index.html,1580672583035,10ff31c184ea6208a1350588a7147c11a92d836d97d8210feeffa4e07b03eb8e
+robots.txt,499162500000,381eb2c6776dd7924af6a6f8761e8720f5c7fb210451f07f2d13cf52af3073d2
+service-worker.js,1580672583035,057466a02cd81b6b02faa9bbf582f1bbba2ed4cb788b0fd56e82187b653e9890
+static/js/2.6b1ddb84.chunk.js.LICENSE,1580672583086,cdcbf976da88c22a204de29bd84862ecb253c8f2a8a7fb3604fda50718f85659
+static/js/main.8a7b8c8d.chunk.js,1580672583059,f6369d1ef1f320e6fcb893bbf1584c3b14c48562568adfc3bdc01c0b824e8f3a
+static/js/runtime-main.60309261.js,1580672583087,a135768f674348ddc5b8955e3966c38c020da16c40cebc059e5aa8a828bbd172
+static/js/runtime-main.60309261.js.map,1580672583087,1d52985e8b129851aec9a18a7d11e6e08bca77617e50f3684df0f246c3daf46d
+static/js/main.8a7b8c8d.chunk.js.map,1580672583085,4a1bf80051fe44b3f95f8bd55d821a81ea9f0d5da9ff638680e9fb99b6fb7789
+static/js/2.6b1ddb84.chunk.js,1580672583086,a3724a2e6d450ddc3d95a742e0255b554f67c9a6cb28ec82e24d6fdf72934526
+static/js/2.6b1ddb84.chunk.js.map,1580672583087,d66c7dcd318109b3efca999ecfd7fecfc31d194d0cdaa4b0ddf98bc8fb1fd277
```
Comment (rota-rossi) [inline .firebase/hosting.YnVpbGQ.cache]: is this cache file supposed to be here?

File: src/views/RegistrationForm/RegistrationForm.tsx [hunk 2]
```diff
@@ -1,13 +1,14 @@
-import React, { useContext } from "react";
+import React, { useContext, useState, useEffect } from "react";
```
Comment (rota-rossi) [inline src/views/RegistrationForm/RegistrationForm.tsx]: Don't use it, lose it.

File: .firebase/hosting.YnVpbGQ.cache (hunk 1 above)
Comment (talyh) [reply .firebase/hosting.YnVpbGQ.cache]: Maybe?
It was generated by firebase deploy when I deploy a new version, and used to calculate the delta for the next version, thus speeding deployment.
I couldn't find anything in firebase's documentation about including or excluding it from source control.

File: src/views/RegistrationForm/RegistrationForm.tsx (hunk 2 above)
Comment (talyh) [reply src/views/RegistrationForm/RegistrationForm.tsx]: 👍 
Will include (well, remove) in the next commit.

**Pull request merged**